src/main/java/nl/dtls/fairdatapoint/api/config/RestApiContext.java

```diff
@@ -62,8 +62,10 @@
 import org.eclipse.rdf4j.model.IRI;
 import org.eclipse.rdf4j.model.ValueFactory;
 import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+import static org.eclipse.rdf4j.rio.RDFFormat.TURTLE;

```

Comment:
Static imports usually go on top of the list. See the convention [here](https://google.github.io/styleguide/javaguide.html#s3.3.3-import-ordering-and-spacing). (although I don't necessarily agree with the "All non-static imports in a single block", I prefer the way eclipse groups them).

Reply:
But netbeans is not grouping them :-( I will modify it manually